{
    "error": {
        "code": "Base.1.1.InternalError",
        "message": "The request failed due to an internal service error.  The service is still operational.",
        "@Message.ExtendedInfo": [
            {
                "@odata.type": "#Message.v1_0_5.Message",
                "MessageId": "Base.1.1.InternalError",
                "Message": "The request failed due to an internal service error.  The service is still operational.",
                "RelatedProperties": [],
                "MessageArgs": [],
                "Severity": "Critical",
                "Resolution": "Resubmit the request.  If the problem persists, consider resetting the service."
            }
        ]
    }
}